src/pages/index.js

```diff
@@ -77,15 +134,28 @@ profileAddButton.addEventListener("click", () => {
 const profileEditPopup = new PopupWithForm(
   {
     popupSelector: "#profile-edit-modal",
-    handleFormSubmit: ({ title, description }) => {
-      profileInfo.setUserInfo(title, description);
-      profileEditPopup.close();
-    }
+    handleFormSubmit: ({ title, description }, profileEditForm) => {
+      profileEditPopup.renderLoading(true);
+      api.updateProfile(title, description)
+        .then((data) => {
+          profileInfo.setUserInfo(data.name, data.about, data.avatar),
```

Comment:
`,` is not needed at the **end** of line 141: it **brakes** the formatting

* You need to format the markup (and code) to show **indentation** and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`



```js
      .then((data) => {
        profileInfo.setUserInfo(data.name, data.about, data.avatar);
        profileEditPopup.close();
      })
      .catch((err) => {
```